staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -48,6 +48,7 @@ import (
 )
 
 const execInfoEnv = "KUBERNETES_EXEC_INFO"
+const maxOnRotateList = 1024 * 1024
```

Comment:
I think we'll run into problems with ephemeral port exhaustion well before we get to a million clients. A process that constructs a thousand instances of a client from a config probably has a problem.

Reply:
also, rename this to make it clearer that the list isn't constrained, it just warns

Reply:
done, thanks